codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -173,32 +166,29 @@ update these file paths accordingly.
     codesignTeamId = availablePasswords['CODESIGN_TEAM_ID']!;
     appSpecificPassword = availablePasswords['APP_SPECIFIC_PASSWORD']!;
 
-    await processRemoteZip();
+    final String? codesignedFilePath = await processRemoteZip();
 
-    if (dryrun) {
-      log.info('code signing dry run has completed, If you intend to upload the artifacts back to'
-          ' google cloud storage, please use the --dryrun=false flag to run code signing script.');
+    if (codesignedFilePath != null) {
+      log.info('code signing dry run has completed, If you have uploaded the artifacts back to'
+          ' google cloud storage, please delete the folder $codesignedFilePath and $inputZipPath');
     }
-    log.info('Codesigned all binaries in ${rootDirectory.path}');
+    log.info('Codesign completed. Codesigned zip is located at ${rootDirectory.path}');
 
-    await rootDirectory.delete(recursive: true);
+    rootDirectory.list(recursive: true).listen((FileSystemEntity file) async {
+      if (file is File && file.path != codesignedFilePath) {
```

Comment:
Is this for extracted paths? if it is I'd recommend to unzip, read, sign and delete temp files from a single try/finally

Reply:
Yep its for cleanup. Sure thanks for the suggestion! I reorganized it in a try finally block.